### PR TITLE
Fixes with UI ux

### PR DESCRIPTION
**Pull request by PanRadaCloudbest**
## What does this PR do?

Fix for crash after putting to the big amount in the receive section
Fix for numeric input has an alphanumeric keyboard

## Todos:

- [x] Checked on iOS
- [ ] Checked on Android

### CODE REVIEW

File: loc/en.js [hunk 1]
```diff
@@ -382,6 +382,7 @@ module.exports = {
       address: 'address',
       address_field_is_not_valid: 'Address field is not valid',
       amount_field_is_not_valid: 'Amount field is not valid',
+      amount_is_too_heigh: "There is a maximum of 21,000,000 BTCV that will ever exist, so don't try to send more!",
```
Comment (marek-siemieniuk-morawski) [inline loc/en.js:385]: it should be `high` instead of `heigh`, I guess

Comment (PanRadaCloudbest) [reply loc/en.js:385]: +1 :)

File: loc/en.js [hunk 2]
```diff
@@ -382,6 +382,7 @@ module.exports = {
       address: 'address',
       address_field_is_not_valid: 'Address field is not valid',
       amount_field_is_not_valid: 'Amount field is not valid',
+      amount_is_too_high: "There is a maximum of 21,000,000 BTCV that will ever exist, so don't try to send more!",
```
Comment (KedziaPawel) [inline loc/en.js:385]: would be nice if 21,000,000 would be as the variable

Comment (PanRadaCloudbest) [reply loc/en.js]: done

**Review by KedziaPawel — APPROVED** [review]

**Pull request merged**